immutrack/src/components/login/LoginChoose.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { connect } from "react-redux";
+import { loginParent, loginProvider } from "../../actions/loginType";
+
+const LoginChoose = ({ history, loginParent, loginProvider }) => {
+  const parent = e => {
+    e.preventDefault();
+    loginParent();
```

Comment:
Where is the return value for the loginParent going to go?

Reply:
I needed a way to switch endpoints for the axios call in Login. Originally, used local storage to set a value based on what button was clicked. Seemed...not good.

This sets the value of `props.user` in  `handleSubmit` in Login.js - via the parent and provider actions in loginType.js. It's hard coded in loginType.js, but if I can figure out what went wrong, I'll switch this back to something like loginProvider({payload: "user"}) to make this more explicit.

Speaking of, it's `props.user` because, for some reason, pulling out `user` from props sets user to undefined instead of parents/providers. I tried pulling user out of props all the way down, but wasn't working.